### DISCUSSION TITLE
interval for dataLabels

### DISCUSSION THREAD

**Discussion by monsterbitar** [nhn/tui.chart#694] [Ideas]
When a graph has data labels, and in particular pretty ones like bubbles, and the number of points in the graph is high, it becomes difficult to use.

I propose adding a new feature, interval for dataLabels such that one can set it to only show every 10th label or some such for highle packed graphs. Ideally, it would have an auto value by default which works similar to the xAxis automatic intervals.

Based on my graphs, I think having twice as many displayed data labels as there are xAxis labels would make sense to me, but anything that's configurable and have some kind of sensible default is great.

**Comment by jwlee1108** [comment]
@monsterbitar 

Thank you for your nice idea. The `dataLabels` issue is something I've been thinking about lately. But I think this post would be better suited to Discussion. Let's move on to Discussion and shape our ideas further.
